src/screens/ContactsList/index.tsx

```diff
@@ -75,6 +82,7 @@ const ContactsListScreen = (): ReactElement => {
             data={contactsSectioned}
             totalElements={contacts.length}
             searchText={searchText}
+            forGroupModeEnabled={!!group}
```

Comment:
@kamil-rusin  Pa na to 😁 Znasz może jakiś lepszy sposób na sprawdzenie, czy zmienna nie jest `undefined` / `null`em / zerem / pustym stringiem / itd?

Reply:
Widziałem parę razy w pracy, ale nie używałem XD Od teraz zacznę 😄